tests/flow/test_entity_update.py

```
@@ -557,3 +558,192 @@ def test38_accumulating_updates(self):
+        self.env.assertEqual(result.properties_set, 2)
+        self.env.assertEqual(result.properties_removed, 3)
+
+        self.env.assertEqual(len(actual_node.properties), 2)
+        self.env.assertEqual(actual_node.properties['a'], 4)
+        self.env.assertEqual(actual_node.properties['c'], 'str')
+
+        #-----------------------------------------------------------------------
+
+        self.graph.delete()
+
+        node  = self.graph.query("CREATE (n {a:1, b:2, c: 'str'}) RETURN n").result_set[0][0]
```

Ruff (0.14.5)

Local variable `node` is assigned to but never used

Remove assignment to unused variable `node`

(F841)